.github/workflows/docs-ci.yaml

```diff
@@ -0,0 +1,54 @@
+jame: documentation CI
+
+on:
```

Comment:
Should we have also `workflow_dispatch` to enable trigger manually?

Reply:
i can add it.
what user input should it take? only `$TAG`?

Reply:
TAG is enough